.github/workflows/php-lint-test.yml

```diff
@@ -0,0 +1,34 @@
+name: PHP linting and tests
```

Comment:
Perhaps we could remove this file for now, and maybe add it in #1425 ? Otherwise these jobs will just fail.

Reply:
Agree, the workflow title is `PHP linting and tests`, but it only contains the `test` job, the `linting` job is being added in [#1425](https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1425). I'll change it to `PHP tests`, and in #1425 change it back